packages/artillery/lib/util.js

```diff
@@ -198,7 +199,7 @@ async function checkConfig(script, scriptPath, flags) {
     );
     payloadSpec.path = resolvedPathToPayload;
   });
-
+  script._scriptPath = absoluteScriptPath;
```

Comment:
Adding the `scriptPath` to the `script` in `checkConfig` function so we can feed it to the `resolveConfigTemplates` from `run.js` to make it available for local runs.

Follows the same process that was done for existing `$dirname` variable.